framework/src/modules/chain/blocks/verify.js

```diff
@@ -491,12 +491,7 @@ class BlocksVerify {
 		);
 	}
 
-	async reloadRequired(blocksCount, memRounds) {
```

Comment:
Can you add some comment here what the goal of this function is?

Reply:
I discussed with @shuse2 before and he said we should probably redefine this function to be more useful/meaningful to BFT